Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Simulation/ScaleFadeAndDestroy.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    public class ScaleFadeAndDestroy : MonoBehaviour
+    {
+        [Tooltip("The value to scale to")]
```

Comment:
This seems like a useful script, but I'm not sure why it is in a PR that adds a touch and grip state to interactable. Did you add it by mistake?

Reply:
This should have been put in the Utilities folder or something, it's a general temporary glow or burst effect. I removed it from this PR.